tests-integration/test-require.spec.ts

```diff
@@ -10,15 +10,6 @@ const NamedAutolinker = require( 'autolinker' ).Autolinker;
 
 describe( 'Autolinker require() tests - ', () => {
 
-	it( `Autolinker should be the default export of 'autolinker'`, () => {
```

Comment:
Hmm, I remember I had to do some hackery to get this to work (might have even been some direct string replacements in the output javascript file.

I'll take a look at this